From 80eb3eda8dd541015bb88b5f64fcf941ad2228e0 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Mon, 27 Aug 2018 14:29:23 +0300
Subject: [PATCH 4453/5242] MLK-18898-3: ARM64: dts: fsl-imx8mq-evk-audio-tdm:
 remove redundant config

commit  7c80c5b175d3c8eab0da44f746ae6cbe836d805c from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove redundant configuration already present in fsl-imx8mq-evk.dts.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit 45545e75ca9cc29418d9674d22c8c6310ccdd0cd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-audio-tdm.dts     |   27 --------------------
 1 file changed, 27 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-audio-tdm.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-audio-tdm.dts
index 698a610..752114f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-audio-tdm.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-audio-tdm.dts
@@ -17,36 +17,9 @@
 / {
 	sound-ak4458 {
 		fsl,tdm;
-		status = "okay";
 	};
 
 	sound-ak5558 {
 		fsl,tdm;
-		status = "okay";
 	};
-
-	sound-ak4497 {
-		status = "disabled";
-	};
-};
-
-&sai1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_sai1>;
-	assigned-clocks = <&clk IMX8MQ_CLK_SAI1_SRC>,
-			<&clk IMX8MQ_CLK_SAI1_DIV>;
-	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <49152000>;
-	status = "okay";
-};
-
-&sai5 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_sai5>;
-	assigned-clocks = <&clk IMX8MQ_CLK_SAI5_SRC>,
-			<&clk IMX8MQ_CLK_SAI5_DIV>;
-	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <49152000>;
-	fsl,sai-asynchronous;
-	status = "okay";
 };
-- 
1.7.9.5

